typegate/core/Cargo.toml

```diff
@@ -5,14 +5,11 @@ version.workspace = true
 
```

Comment:
I would suggest to rename typegate/core into typegate/engine and the crate `typegate_engine`, see after for the "standalone" version

Reply:
Good names. `typegate_core` is too door `typehraph_core`